docs/starters.yml

```diff
@@ -5199,3 +5199,14 @@
     - Disqus
     - Resume
     - Place plan on the top
+- url: https://gatsby-portfolio-starter.netlify.com/
+  repo: https://github.com/Judionit/gatsby-portfolio-starter
+  description: A simple Gatsby portfolio starter
+  tags:
+    - Netlify
+    - Styling: CSS-in-JS
```

Comment:
Can you please change this tag from `Styling: CSS-in-JS` to `Styling:CSS-in-JS` such that it reflects the correct tag?